### ISSUE TITLE
Not block incoming SMS

### ISSUE THREAD

**Issue by m-ebrahimi** [subprogram/ParanoidSmsBlocker#4]
Hi,
I install APP on my phone. I defined the black list. but I see that the APP show a notification for blocking a SMS and also my phone show that I have a new SMS!

Sony Xperia Z
Android 4.4.4

Thanks

**Comment by subprogram** [comment]
Yes, it doesn't work on Android 4.4+ because Google made some changes in Android 4.4 APIs.
For more information: 
https://developer.android.com/about/versions/android-4.4.html
http://android-developers.blogspot.ru/2013/10/getting-your-sms-apps-ready-for-kitkat.html

**Comment by m-ebrahimi** [comment]
Can you apply some change for working with android 4.4+ with this solution?

1) User's change default SMS app to ParanoidSmsBlocker.
2) ParanoidSmsBlocker filter SMS and block some SMS but the white list SMS saved to user's Inbox (native SMS App).
3)User can delete and manage messages in native SMS app.

I think this App do this :
https://play.google.com/store/apps/details?id=com.visinor.phonewarrior&hl=en

Thanks.

**Comment by subprogram** [comment]
Unfortunately, it is not so easy... User should also use ParanoidSmsBlocker to sending SMS if he changed default SMS app to it.
I don't know how Phone Warrior do this and I haven't got a device with android 4.4+ for testing. But if you can write patch, I'll gladly merge it :)
